### DISCUSSION TITLE
MIT MOOSE workshop registration

### DISCUSSION THREAD

**Discussion by chenzhao-py** [idaholab/moose#26523] [Q&A General]
Hey Guys,

Can anyone help me to check if I got registered successfully? I wanted to attend the workshop on Jan16th but I am not sure if I should have received another confirmation email. Below is the message that I got:

_Thank you for your interest in MOOSE Framework Fundamentals@MIT January 2024 scheduled for 01/16/2024 - 01/17/2024.

We have received your registration request and will pass this information on to the course coordinators.

Best Regards,

NCRC Staff_

Chen

**Comment by GiudGiud** [comment]
Hello

Yes I have you in the list. 
I must have sent the first information email just before you registered. 
I will send an email again about the training soon

Guillaume

**Reply by hardud** [reply]
I am in the same boat. I have the same message, nothing more.
Can you send me the email about training as well? 
Thanks,
Harsh Dudhatra

**Reply by GiudGiud** [reply]
Hi Harsh can you send me your email please? You should be in I just didnt have you in the list when i sent the follow up

**Reply by hardud** [reply]
My email is hpd26@miami.edu

One of my batch mate, Abdulmohssen also registered for the workshop and didn't receive the conformational email.
His email id is: a.abaalkhail@umiami.edu

Thanks,
Harsh
